utils/sources.rb

```diff
@@ -61,24 +69,30 @@ def to_s
 end
 
 opts = GetoptLong.new(
-  [ '--help', '-h', GetoptLong::NO_ARGUMENT ],
-  [ '--url' , '-u', GetoptLong::NO_ARGUMENT ],
+  [ '--help'    , '-h', GetoptLong::NO_ARGUMENT       ],
+  [ '--url'     , '-u', GetoptLong::NO_ARGUMENT       ],
+  [ '--mirror'  , '-m', GetoptLong::REQUIRED_ARGUMENT ],
 )
 
 
 url_only = false
+mirror   = nil
 
 opts.each do |opt, arg|
   case opt
     when '--help'
       puts <<-EOF
       sources.rb [option] path/package/url ..
 
-      --url, -u:    just emit URLs
+      --url, -u                               just emit URLs
+      --mirror http://example.com/some/path   download from mirror
+      --help                                  show this help
```

Comment:
This is a really minor thing, but I was wondering whether the shorthand `-m` and `-h` switches for `--mirror` and `--help` are deliberately omitted from here?  (Although I'm not sure how we could list both switches for `--mirror` as it has an argument as well.)